daemon/api.go

```diff
@@ -172,6 +173,12 @@ var (
 		UserOK: false,
 		POST:   postCreateUser,
 	}
+
+	buyCmd = &Command{
+		Path:   "/v2/buy",
+		UserOK: true,
```

Comment:
We don't want UserOK here.. the name is ambiguous, sorry. It actually means "local system users that are not authenticated".